include/rotatingMHD/convergence_test.h

```diff
@@ -147,8 +161,77 @@ struct ConvergenceTestParameters
 template<typename Stream>
 Stream& operator<<(Stream &stream, const ConvergenceTestParameters &prm);
 
+
+/*!
+ * @class ConvergenceTestData
+ *
+ * @brief @ref ConvergenceTestData is a book-keeping class for the errors of a convergence test.
+ *
+ */
+class ConvergenceTestData
+{
+
+public:
+
```

Comment:
Documentation

Reply:
Constructor doc string is still missing in 3536dc4.